src/themes/teams-high-contrast/siteVariables.ts

```diff
@@ -1,10 +1,17 @@
-import { white } from '../teams/siteVariables'
+import { white, green } from '../teams/siteVariables'
 
 //
 // COLORS
 //
 export const black = '#000'
 
+export const hyperlinkColor = '#ffff01'
+export const disabledColor = '#3ff23f'
+export const selectionBackgroundColor = '#1aebff'
```

Comment:
these may be older colors or maybe I have older color redlines... (sending image via teams)